pyanaconda/argument_parsing.py

```diff
@@ -541,25 +540,29 @@ def __call__(self, parser, namespace, values, option_string=None):
     class ParseNosave(Action):
         def __call__(self, parser, namespace, values, option_string=None):
             options = []
+            namespace.copy_input_kickstart = None
+            namespace.save_output_kickstart = None
+            namespace.save_logs = None
             if values:
                 options = values.split(",")
             if "all" in options:
-                flags_instance.nosave_input_ks = True
-                flags_instance.nosave_output_ks = True
-                flags_instance.nosave_logs = True
+                namespace.copy_input_kickstart = False
+                namespace.save_output_kickstart = False
+                namespace.save_logs = False
             else:
                 if "all_ks" in options:
-                    flags_instance.nosave_input_ks = True
-                    flags_instance.nosave_output_ks = True
+                    namespace.copy_input_kickstart = False
+                    namespace.save_output_kickstart = False
                 else:
                     if "input_ks" in options:
-                        flags_instance.nosave_input_ks = True
+                        namespace.copy_input_kickstart = False
                     if "output_ks" in options:
-                        flags_instance.nosave_output_ks = True
+                        namespace.save_output_kickstart = False
                 if "logs" in options:
-                    flags_instance.nosave_logs = True
+                    namespace.save_logs = False
 
-    ap.add_argument("--nosave", action=ParseNosave, nargs="?", help=help_parser.help_text("nosave"))
+    ap.add_argument("--nosave", action=ParseNosave, nargs="?", default="",
```

Comment:
Could you try to use `SUPPRESS` and `if name in opts`? Then you don't have to set the values to `None`. I guess we should start to go in that direction. See: https://github.com/rhinstaller/anaconda/pull/3561/files

Reply:
`SUPPRESS` will not do anything because we add these values in a custom action - if it isn't invoked, the args are not there anyway. The rest is correct though, let's try this.